pom.xml

```diff
@@ -1,357 +1,371 @@
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-source-plugin</artifactId>
-				<version>3.1.0</version>
-				<executions>
-					<execution>
-						<id>bundle-sources</id>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>3.1.1</version>
-				<executions>
-					<execution>
-						<id>bundle-javadoc</id>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
-				<configuration>
-					<!-- downgrade to fix Maven build with Java 13 -->
-					<!-- see https://bugs.openjdk.java.net/browse/JDK-8212233 -->
-					<source>8</source>
-				</configuration>
-			</plugin>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-shade-plugin</artifactId>
-				<version>3.2.1</version>
-				<executions>
-					<execution>
-						<phase>package</phase>
-						<goals>
-							<goal>shade</goal>
-						</goals>
-						<configuration>
-							<createDependencyReducedPom>false</createDependencyReducedPom>
-							<minimizeJar>true</minimizeJar>
-							<shadedArtifactAttached>true</shadedArtifactAttached>
-							<shadedClassifierName>jar-with-dependencies</shadedClassifierName>
-							<transformers>
-								<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
-									<manifestEntries>
-										<Implementation-Vendor-Id>${project.groupId}</Implementation-Vendor-Id>
-										<Implementation-Title>${project.artifactId}</Implementation-Title>
-										<Implementation-Version>${project.version}</Implementation-Version>
-									</manifestEntries>
-								</transformer>
-							</transformers>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-
-			<plugin>
-				<groupId>org.sonatype.plugins</groupId>
-				<artifactId>nexus-staging-maven-plugin</artifactId>
-				<version>1.6.8</version>
-				<extensions>true</extensions>
-				<configuration>
-					<nexusUrl>https://oss.sonatype.org/</nexusUrl>
-					<serverId>ossrh</serverId>
-					<stagingProfileId>506dfd640832af</stagingProfileId>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-
-	<profiles>
-		<profile>
-			<id>sign</id>
-			<build>
-				<plugins>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-gpg-plugin</artifactId>
-						<version>1.6</version>
-						<configuration>
-							<skip>${gpg.skip}</skip>
-							<passphrase>${env.GPG_PASSPHRASE}</passphrase>
-						</configuration>
-						<executions>
-							<execution>
-								<id>sign-artifacts</id>
-								<phase>verify</phase>
-								<goals>
-									<goal>sign</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
-</project>
+<?xml version="1.0"?>

```

Comment:
Why this massive diff? Did you make any whitespace or similar changes? Other classes are affected too (e.g. `PageFrameIT`).

Reply:
I compared the ``pom`` file and also ``PageFrameIT`` but could not see such a huge diff. Might this be related to Linux / Windows changes of tabs, spaces or line break? How does the diff look on Linux?

Reply:
This is the commit by you:

https://github.com/retest/recheck-web/commit/ae3a317c5ba5544413ea40600c5e0e150ad0fd33#diff-600376dffeb79835ede4a0b285078036

I assume this is related to some indentation, line endings, … changes since all lines differ.

Reply:
I accidentally did not convert line endings in commit ae3a317 from windows to unix style. For pom.xml, this got fixed in a7920537. The last remaining file touched in this PR which got committed with wrong line endings was WebDriverFactory.java. I fixed the line endings in this file also.